x/pool-incentives/keeper/grpc_query_test.go

```diff
@@ -443,6 +443,7 @@ func (s *KeeperTestSuite) TestExternalIncentiveGauges_NoLock() {
 		defaultNoLockGaugeConfig = gaugeConfig{
 			distributeTo: lockuptypes.QueryCondition{
 				LockQueryType: lockuptypes.NoLock,
+				Duration:      time.Nanosecond,
```

Comment:
nit: can we define a variable somewhere such as `DefaultNoLockDuration` so that if we need to change it from `1ns`, we only need to change it in one place

Reply:
Great point. Added 00b845b3320834dfd410f8d06d5e5df1678ce346